workbase/src/renderer/components/Visualiser/TopBar/GraqlEditor/GraqlEditor.vue

```diff
@@ -217,6 +217,12 @@
     },
     created() {
       this.renderIcons();
+
+      window.addEventListener('keydown', (e) => {
+        const gKey = 71;
```

Comment:
Can we move this const at the top of the file? 
If you declare it inside the function it gets re-declared every time the keydown event is triggered.
Also let's use the convention for global constants, which is all CAPS, smth like: 
`const LETTER_G_KEYCODE = 71`

Reply:
I also moved it to the visualiser store